src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildWorkspaceUpgradeContext.cs

```diff
@@ -224,6 +225,8 @@ private void Workspace_WorkspaceFailed(object? sender, WorkspaceDiagnosticEventA
 
         public IUpgradeContextProperties Properties { get; }
 
+        public ICollector<AnalyzeResultDefinition> Results { get; } = new ListCollector<AnalyzeResultDefinition>();
```

Comment:
I'm surprised - is this _really_ the _only_ place we implement ``IUpgradeContext`?? Feels unnatural that it's an MsBuild Workspace context but this is the thing we pass in/out of upgraders, etc. but I digress